code-challenges.js

```diff
@@ -75,33 +84,42 @@
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// Create a const "reverse" = .split "currentCohort"
```

Comment:
What is your current output when you run this?  Split is a good start to this problem, however, split will turn your string into an array, there are a few additional steps needed to reverse this.